android-o/AndroidPlayLocation/LocUpdPendIntent/LocUpdPendIntent/Utils.cs

```diff
@@ -74,8 +77,22 @@ public static void SendNotification(Context context, string notificationDetails)
 	        // Get an instance of the Notification manager
 			var mNotificationManager = context.GetSystemService(Context.NotificationService) as NotificationManager;
 
-			// Issue the notification
-			mNotificationManager.Notify(0, builder.Build());
+		    // Android O requires a Notification Channel.
+		    if (Build.VERSION.SdkInt>= Build.VERSION_CODES.O)
+		    {
+		        var name = (Java.Lang.String)context.GetString(Resource.String.app_name);
```

Comment:
This cast throws an exception on startup on Oreo devices. I'd suggest simply using
>  string name = context.GetString(Resource.String.app_name);

Reply:
Changed.